ui/raidboss/data/06-ew/raid/p5s.ts

```diff
@@ -1,13 +1,212 @@
+          return;
+
+        if (data.target === data.me)
+          return output.busterOnYou!();
+        return output.busterOnTarget!({ player: data.ShortName(data.target) });
+      },
+      outputStrings: {
+        busterOnYou: Outputs.tankBusterOnYou,
+        busterOnTarget: Outputs.tankBusterOnPlayer,
+        tankSwap: Outputs.tankSwap,
+      },
+    },
+    {
+      id: 'P5S Toxic Crunch',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '784A', source: 'Proto-Carbuncle', capture: false }),
+      infoText: (data, _matches, output) => {
+        if (data.role === 'tank' && data.target !== data.me)
+          return;
+
+        if (data.target === data.me)
+          return output.busterOnYou!();
+        return output.tankBuster!();
+      },
+      outputStrings: {
+        busterOnYou: Outputs.tankBusterOnYou,
+        tankBuster: Outputs.tankBuster,
+      },
+    },
+    {
+      id: 'P5S Double Rush',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '771B', source: 'Proto-Carbuncle', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P5S Venom Squall/Surge',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '771[67]', source: 'Proto-Carbuncle' }),
+      durationSeconds: 5,
+      alertText: (_data, matches, output) => {
+        const spread = output.spread!();
+        const healerGroups = output.healerGroups!();
+        // Venom Squall
+        if (matches.id === '7716')
+          return output.text!({ dir1: spread, dir2: healerGroups });
+        return output.text!({ dir1: healerGroups, dir2: spread });
+      },
+      outputStrings: {
+        healerGroups: Outputs.healerGroups,
+        spread: Outputs.spread,
+        text: {
+          en: '${dir1} -> Bait -> ${dir2}',
+        },
+      },
+    },
+    {
+      id: 'P5S Tail to Claw',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7712', source: 'Proto-Carbuncle', capture: false }),
+      durationSeconds: 5,
+      response: Responses.getFrontThenBack(),
+    },
+    {
+      id: 'P5S Claw to Tail',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '770E', source: 'Proto-Carbuncle', capture: false }),
+      durationSeconds: 5,
+      response: Responses.getBackThenFront(),
+    },
+    {
+      id: 'P5S Topaz Ray Collect',
+      type: 'StartsUsing',
+      // 7703: 3.7s, 7704: 6.2s, 7705: 8.7s, 7706: 11.2s
+      netRegex: NetRegexes.startsUsing({ id: '770[3456]', source: 'Proto-Carbuncle' }),
+      run: (data, matches) => {
+        const index = convertAbilityIdToTopazRayIndex(matches.id);
+        if (data.topazRays[index] === undefined)
+          data.topazRays[index] = [];
+        const direction = convertCoordinatesToDirection(parseFloat(matches.x), parseFloat(matches.y));
+        data.topazRays[index]?.push(direction);
+      },
+    },
+    {
+      id: 'P5S Topaz Ray',
```

Comment:
How confident are you on this one? Is it worth landing the more sure stuff first and then landing this later after direct testing?

Reply:
As sure as I can be without video, honestly. The logic is pretty sound and it gives 4 results, but I can't confirm them otherwise. Is there a way to make it experimental?

Reply:
No, just remove it and add it locally and then send another PR later.